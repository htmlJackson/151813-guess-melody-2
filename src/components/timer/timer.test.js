import React from "react";
import renderer from "react-test-renderer";

import Timer from "../timer/timer.jsx";

describe(`Проверка <Timer>`, () => {
  it(`компонент <Timer> корректно отрисован`, () => {

    const tree = renderer.create(<Timer gameTime={5} onTimerTick={jest.fn()} onTimerExpire={jest.fn()} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
