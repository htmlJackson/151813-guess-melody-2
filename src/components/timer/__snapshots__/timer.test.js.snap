// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверка <Timer> компонент <Timer> корректно отрисован 1`] = `
<div
  className="timer__value"
  xmlns="http://www.w3.org/1999/xhtml"
>
  <span
    className="timer__mins"
  >
    0
  </span>
  <span
    className="timer__dots"
  >
    :
  </span>
  <span
    className="timer__secs"
  >
    05
  </span>
</div>
`;
